README.md

```diff
@@ -28,14 +28,14 @@ Rust.
 Documentation
 -------------
 
-The current implementation supports SPIR-V 1.4 (Revision 1).
+The current implementation supports SPIR-V 1.5 (Revision 4).
```

Comment:
I really am not sure about this. https://www.khronos.org/registry/spir-v/ mentions `Revision 5` but we're on the `1.5.4.raytracing.fixed` tag. There is no `1.5.5` tag.